schema/features/fingerprinting-canvas.ts

```diff
@@ -1,7 +1,10 @@
 import { Feature, CSSInjectFeatureSettings, CSSConfigSetting, FeatureState } from '../feature';
 
 type FullFingerprintingCanvasOptions = CSSInjectFeatureSettings<{
+    // Codebase change
     additionalEnabledCheck?: FeatureState;
```

Comment:
This we already had and it's a different name. I'll remove it soon as we've stopped using it for an experiment.